### PR TITLE
Update running cljdoc locally page: dependencies

### LINKED ISSUE #381: DOC Dependency to CLI tool
I'm still a beginner in the Clojure world, and I was following the setup for [Running cljdoc locally](https://github.com/cljdoc/cljdoc/blob/master/doc/running-cljdoc-locally.md). When I arrived at the command to launch the cljdoc server:
```sh
./script/cljdoc run
```
I got this weird error message:
```
Exception in thread "main" java.io.FileNotFoundException: -A:cli (Aucun fichier ou dossier de ce type)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(FileInputStream.java:195)
	at java.io.FileInputStream.<init>(FileInputStream.java:138)
	at java.io.FileInputStream.<init>(FileInputStream.java:93)
	at clojure.lang.Compiler.loadFile(Compiler.java:7449)
	at clojure.main$load_script.invokeStatic(main.clj:278)
	at clojure.main$script_opt.invokeStatic(main.clj:338)
	at clojure.main$script_opt.invoke(main.clj:333)
	at clojure.main$main.invokeStatic(main.clj:424)
	at clojure.main$main.doInvoke(main.clj:387)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.lang.Var.applyTo(Var.java:702)
	at clojure.main.main(main.java:37)
```
After a while, I discover that I didn't use the CLI tool which is required to get cljdoc running.

This dependency should be explicit in the documentation page "Running cljdoc locally". As the [official documentation](https://clojure.org/guides/getting_started) which describes its installation is straightforward, a simple link to it will get the job done.

### PR DESCRIPTION
**Pull request by frozar**
Add a section 'Dependencies cljdoc' in the documentation page 'running-cljdoc-locally.md'.

Fixes #381

### CODE REVIEW

File: doc/running-cljdoc-locally.md [hunk 1]
```diff
@@ -10,6 +10,15 @@ to instead [run cljdoc locally from docker](#run-cljdoc-locally-from-docker).
 
 ## Run cljdoc Locally from Source
 
+### Dependencies cljdoc
+
+To run cljdoc from Source, you must install these dependencies:
```
Comment (martinklepsch) [inline doc/running-cljdoc-locally.md:15]:
```suggestion
To run cljdoc from source, you must install these dependencies:
```

File: doc/running-cljdoc-locally.md [hunk 2]
```diff
@@ -10,6 +10,15 @@ to instead [run cljdoc locally from docker](#run-cljdoc-locally-from-docker).
...
+ - nodejs
+ - clojure and its command line tools
+
+To install nodejs, follow [instructions here](https://nodejs.org/en/).
+To install clojure and tools, follow [instructions here](https://clojure.org/guides/getting_started).
+
```
Comment (martinklepsch) [inline doc/running-cljdoc-locally.md:21]: To keep this dense maybe you can combine the list of dependencies with the installation instructions? E.g.

- nodejs ([installation instructions](https://nodejs.org/en/))
- Clojure CLI tools ([installation instructions](https://clojure.org/guides/getting_started))

```md
- nodejs ([installation instructions](https://nodejs.org/en/))
- Clojure CLI tools ([installation instructions](https://clojure.org/guides/getting_started))
```

File: doc/running-cljdoc-locally.md (hunk 1 above)
Comment (frozar) [reply doc/running-cljdoc-locally.md]: I agree, taken into account in 25db0ff

File: doc/running-cljdoc-locally.md (hunk 2 above)
Comment (frozar) [reply doc/running-cljdoc-locally.md:18]: I agree, taken into account in 25db0ff

**Comment by martinklepsch** [comment]
Awesome, thank you very much 🙌

**Pull request merged**